exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -50,21 +50,19 @@ type OperationMap = Map<number, Operation>;
 type OptimisticDependencies = Map<number, Dependencies>;
 type DependentOperations = Record<string, number[]>;
 
-export interface CacheExchangeOpts {
-  updates?: Partial<UpdatesConfig>;
-  resolvers?: ResolverConfig;
-  optimistic?: OptimisticMutationConfig;
-  keys?: KeyingConfig;
-  schema?: IntrospectionData;
-  storage?: StorageAdapter;
-}
-
-export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
-  forward,
-  client,
-  dispatchDebug,
-}) => {
-  const store = new Store(opts);
+export type CacheExchangeOpts = {
+  updates: Partial<UpdatesConfig>;
```

Comment:
Let's remove `Partial` here and instead make the internal `UpdatesConfig` type partial? We can still "unpartial" it on the store

Reply:
It's an interface as we're extending from it so that's not possible.